content/chainguard/administration/terraform-provider/index.md

```diff
@@ -0,0 +1,212 @@
+  }
+}
+```
+
+If you don't have an active Chainguard token when you apply the configuration, the provider will automatically launch a browser to complete the Oauth2 flow with one of the default identity providers: GitHub, GitLab, or Google. This means the Terraform provider does not require `chainctl` to be installed to manage authentication with Chainguard.
+
+You can customize the behavior of the authentication flow in a number of ways. For example, you can specify an identity to assume or a [verified organization](/chainguard/administration/iam-groups/verified-orgs/) name in order to use a previously-configured custom identity provider:
+
+```
+provider "chainguard" {
+  login_options {
+	organization_name = "my-org.com"
+	# identity_id is the exact ID of an assumable identity.
+	# Get this ID with chainctl iam identities list
+	identity_id = "1f127a7c0609329f04b43d845cf80eea4247a07c/d6305475446bbef6"
+  }
+}
+```
+
+You can also configure the provider to use an OIDC token, either directly supplied or from a file, with the automatic browser flow disabled. This is useful when setting up CI workflows:
+
+```
+provider "chainguard" {
+  login_options {
+	# Disable the automatic browser authentication flow.
+	disabled   	= true
+	identity_token = "/path/to/oidc/token"
+  }
+}
+```
+
+You can find more information about authenticating with the Chainguard Terraform provider in the [provider documentation](https://registry.terraform.io/providers/chainguard-dev/chainguard/latest/docs) and [the included examples](https://github.com/chainguard-dev/terraform-provider-chainguard/tree/main/examples).
+
+## Defining Organization References
+
+Terraform was designed to allow users to manage various resources declaratively. In practice, this means that you define the state you want your resources to be in and then you let Terraform and the provider handle the details of bringing that state to reality. 
+
+Resources on the Chainguard platform are organized in a hierarchical structure of folder-like groups, with an organization's group at the root, though most users will only need to work at the organization level.
+
+![Diagram outlining hierarchical structure of Chainguard resources. The diagram has two halves: one labeled "Organization" and another labeled "Chainguard". Under Organization is a box labeled "Tags" with an arrow pointing toward another box labeled "Repos." Under both halves is a box labeled "Role Bindings" which has four arrows pointing from it. Two arrows point to boxes (labeled "Identities" and "Custom Roles") under Organization and the other two point to boxes (labeled "User Identities" and "roles) under Chainguard.](tf-diagram.png)
+
+All user-managed resources are defined in relation to some parent group. This means developers need to be able to reference their organization's group throughout their configuration. We'll outline two ways to define this kind of reference: using local values and data sources.
```

Comment:
Maybe clarify this a bit if the parent group is always going to be the organization group

Reply:
added a note clarifying this point